### DISCUSSION TITLE
Better gates

### DISCUSSION THREAD

**Discussion by Scwestbrook** [ChAoSUnItY/EkiLib#6] [Ideas]
Hi, I've noticed that the gates stay open for a set amount of time, and this has been really annoying as other players just walk in behind me. Could you add a feature, where in addition to the time, if another player approaches the gate it closes? Thanks for your time!

**Comment by ChAoSUnItY** [comment]
I think adds a config option to adjust the time gate opens is much matching your idea?

**Reply by Scwestbrook** [reply]
No, I was thinking more where the gates stay open until a player walks through, and close __IMMEDIATELY__ afterward, because other players have been following me in when I use my ticket.
